js/imdb.service.js

```diff
@@ -1,2 +1,15 @@
 // add a service here that makes a call to the imdb250.json file
 // this service will be utilized by your list controller
+angular.module('services', [])
+
+angular.module('services').factory('imdbService', function($http) {
+	return {
+		getImdbJson: function() {
+			return $http.get('/data/imdb250.json');
+		},
+
+		getOmdbDetails: function(id) {
```

Comment:
You went above and beyond using the omdb api! Excellent! What data are you loading from omdb?